VAMobile/src/screens/HealthScreen/CernerAlert/CernerAlert.tsx

```diff
@@ -83,13 +83,15 @@ function CernerAlert() {
   }
 
   return (
-    <CollapsibleAlert
-      border="warning"
-      headerText={headerText}
-      body={accordionContent()}
-      a11yLabel={headerA11yLabel}
-      onExpand={() => logAnalyticsEvent(Events.vama_cerner_alert_exp())}
-    />
+    <AlertWithHaptics
+      variant="warning"
+      expandable={true}
+      focusOnError={false}
```

Comment:
At first, I was going to suggest adding logic to the `AlertWithHaptics` component that sets `focusOnError` to `false` by default when `exxpandable` is `true` since the `CollapsibleAlert` didn't ever focus on error, but I actually like having the flexibility to be able to set `focusOnError` even for expandable alerts. We don't do this anywhere yet, but it's nice to be able to do that if we need to